src/Engine.ts

```diff
@@ -248,17 +269,7 @@ export function getCardAttribute(
         targetBoardCardID
       );
 
-      const targetCount =
-        action.TargetCount == null
-          ? targetCards.length
-          : getActionValue(
-              gameState,
-              action.TargetCount,
-              playerID,
-              boardCardID,
-              targetPlayerID,
-              targetBoardCardID
-            );
```

Comment:
I don't think this is safe to delete. In V2_Cards.json, there are 10 instances of

```
          "TargetCount": {
            "$type": "TFixedValue",
            "Value": 1
          },
```

Reply:
Nevermind, I misread, this is in the Aura handling, indeed they don't have any that I can find.